alf/experience_replayers/experience_replay.py

```diff
@@ -59,8 +59,13 @@ def replay(self, sample_batch_size, mini_batch_length):
             mini_batch_length (int): the temporal length of each sample
 
         Output:
-            exp (Experience): each item has the shape (`sample_batch_size`,
-                `mini_batch_length`, ...)
+            tuple:
+                - nested Tensors: The samples. Its shapes are [batch_size, batch_length, ...]
```

Comment:
nested Tensors -> Experience

Reply:
It doesn't have to be Experience. User can decide to put any kind of data to the replay buffer.